config/stage.json

```diff
@@ -72,6 +72,7 @@
 		"post-editor/html-toolbar": true,
 		"post-editor/insert-menu": true,
 		"post-editor/image-editor": true,
+		"post-editor/mentions": true,
```

Comment:
Usually we add development as early as possible in the process; open up in wpcalypso as soon as it starts taking shape; horizon if we want to do outside user-testing; and stage / prod as part of launching a feature.